src/pages/components/components.json

```
@@ -1,4 +1,15 @@
 [
+  {
+    "title: SCR Svelte Client Router",
```

There are missing quotes (`"`) here
```suggestion
    "title": "SCR Svelte Client Router",
```